client/checkout/upe-styles/index.js

```diff
@@ -4,20 +4,204 @@
 import { upeRestrictedProperties } from './upe-styles';
 import { generateHoverRules, generateOutlineStyle } from './utils.js';
 
+const appearanceSelectors = {
+	default: {
+		hiddenContainer: '#wcpay-hidden-div',
+		hiddenInput: '#wcpay-hidden-input',
+		hiddenInvalidInput: '#wcpay-hidden-invalid-input',
+	},
+	classicCheckout: {
+		appendTarget: '.woocommerce-billing-fields__field-wrapper',
+		upeThemeInputSelector: '#billing_first_name',
+		upeThemeLabelSelector: '.woocommerce-checkout .form-row label',
+		rowElement: 'p',
+		validClasses: [ 'form-row' ],
+		invalidClasses: [
+			'form-row',
+			'woocommerce-invalid',
+			'woocommerce-invalid-required-field',
+		],
+	},
+	blocksCheckout: {
+		appendTarget: '#shipping.wc-block-components-address-form',
+		upeThemeInputSelector: '#shipping-first_name',
```

Comment:
@FangedParakeet I believe you don't have shipping methods configured on WooCommerce. It looks like if shipping methods are present, WC Blocks displays shipping fields and if not, billing fields are displayed.

I haven't noticed this until now. Since blocks use react, the fields are added to the DOM on request, so we can't stick to a single selector. I'll add a check to find which fields are active on the page before generating the styles.